doc/Developing.md

```diff
@@ -50,7 +50,16 @@ tab with the frontend.
 
 ## Run the electron-based example application
 
-It can also be started with:
+From the root directory run:
+
+    npm run rebuild:electron
+
+This command rebuilds native node packages against the version of node used by electron.
+To rollback changes run:
+
+    npm run rebuild:web
+
+It can also be started with from examples/electron directory:
```

Comment:
After rebuilding for electron, web won't work, but by default web is working. I can put in the web section that if you run `npm run rebuild:electron` don't forget to run `npm run rebuild:web` fisrt.